spec/src/main/asciidoc/chapters/language/expressions.adoc

```diff
@@ -423,13 +423,32 @@ Within the pattern, `_` matches any single character, and `%` matches any sequen
 
 The equality and inequality operators are `=`, `&lt;&gt;`, `&lt;`, `&gt;`, `&lt;=`, `&gt;=`.
 
+The operands of an equality or inequality operator must have the same type,
+ and it must be an atomic type considered _comparable_ by the implementation of Jakarta Query.
+Any type with a <<natural-order,natural order>> must be treated as a comparable type, and the equality and inequality operators must be interpreted in a way which is consistent with the natural order.
+
+[cols="15,~"]
+|===
+| Operator | Interpretation in terms of natural order
+
+| `x < y` | Satisfied if `x` occurs before `y` in the natural order
+| `x > y` | Satisfied if `x` occurs after `y` in the natural order
+| `x <= y` | Satisfied if `x>y` is not satisfied
+| `x >= y` | Satisfied if `x<y` is not satisfied
```

Comment:
Similarly here, if `5 > null` is not satisfied, that should not imply that `5 <= null` (except for if `NULLS LAST` is used)

If the language from my other comment is acceptable, we could do this:
```suggestion
| `x <= y` | Satisfied if `x` occurs before or at identical position to `y` in the natural order
| `x >= y` | Satisfied if `x` occurs after or at identical position to `y` in the natural order
```